src/test/java/com/manywho/services/sql/utilities/DbTestConfigurationProperties.java

```diff
@@ -10,8 +10,8 @@ public DbTestConfigurationProperties() {
         try {
             properties = new Properties();
 
-            if(getClass().getClassLoader().getResourceAsStream("configuration.properties") != null) {
-                properties.load(getClass().getClassLoader().getResourceAsStream("configuration.properties"));
+            if(getClass().getClassLoader().getResourceAsStream("configuration.properties.dist") != null) {
```

Comment:
Is this on purpose? Feels like it should be using the non `.dist` one

Reply:
yes I don't know if it is worth it to keep it, I did it before because the travis configuration. What do you think about get rid of the getenv and the .dist and keep only the configuration.properties file?